hbase-shell/src/main/ruby/shell/commands/clear_deadservers.rb

```diff
@@ -35,18 +37,20 @@ def help
       end
 
       # rubocop:disable Metrics/AbcSize
-      # rubocop:disable Metrics/MethodLength
       def command(*dead_servers)
         servers = admin.clear_deadservers(dead_servers)
         if servers.size <= 0
           formatter.row(['true'])
+          []
         else
           formatter.row(['Some dead server clear failed'])
           formatter.row(['SERVERNAME'])
-          servers.each do |server|
-            formatter.row([server.toString])
+          server_names = servers.map { |server| server.toString }
```

Comment:
Bug fix?

Reply:
This is just a change to collect all the server names as an array so that we can both print and return the list of servers that could not be cleared